hudi-client/hudi-java-client/src/test/java/org/apache/hudi/common/table/read/TestHoodieFileGroupReaderOnJava.java

```diff
@@ -60,8 +60,6 @@ public void assertRecordMatchesSchema(Schema schema, IndexedRecord record) {
 
   @Override
   public HoodieTestDataGenerator.SchemaEvolutionConfigs getSchemaEvolutionConfigs() {
-    HoodieTestDataGenerator.SchemaEvolutionConfigs configs = new HoodieTestDataGenerator.SchemaEvolutionConfigs();
-    configs.addNewFieldSupport = false;
-    return configs;
+    return new HoodieTestDataGenerator.SchemaEvolutionConfigs();
```

Comment:
nit: Could we make this the default implementation in the base class `TestHoodieFileGroupReaderBase`, and only override for Flink, which does not support some evolution?